main.py

```diff
@@ -101,16 +129,13 @@ def get_parameters():
     return _args
 
 
-def handler(event, context):
-    args = get_parameters()
-    logger.info(args)
+args = get_parameters()
+setup_logging(logger_level)
```

Comment:
I think you need to assign this to `logger` e.g.
```suggestion
logger = setup_logging(logger_level)
```

Reply:
Resolved